packages/styleUtils/typography/stories/text.stories.tsx

```diff
@@ -16,23 +10,7 @@ import {
   Text,
   WarningText
 } from "../";
-
-const colors = [
-  themeTextColorPrimary,
-  themeTextColorSecondary,
-  themeTextColorDisabled,
-  blue
-];
-
-const textAlign = [
-  "center",
-  "end",
-  "justify",
-  "left",
-  "match-parent",
-  "right",
-  "start"
-];
+import { textAlignValues } from "../../../storybookHelpers/controlContants";
 
 export default {
   title: "Typography/Text",
```

Comment:
It appears that the control for text color is an object so we could improve these controls by including the `{type: color}` here as well

Reply:
Good shout. Do you think it's worth disabling the `color` control for the stories where a color is set by the component, for instance `Warning` and `Success`?

Reply:
Yes, good point, was going to mention that as well. Should disable color for Warning/Success to suggest that it shouldn't really be changed.